internal/server/get_payments.go

```diff
@@ -12,16 +12,19 @@ type GetPaymentsClient interface {
 	RefDataByCategory(ctx sirius.Context, category string) ([]sirius.RefDataItem, error)
 	Payments(ctx sirius.Context, id int) ([]sirius.Payment, error)
 	Case(sirius.Context, int) (sirius.Case, error)
+	GetUserDetails(sirius.Context) (sirius.User, error)
 }
 
 type getPaymentsData struct {
 	XSRFToken string
 
-	Case           sirius.Case
-	Payments       []sirius.Payment
-	PaymentSources []sirius.RefDataItem
-	ReferenceTypes []sirius.RefDataItem
-	TotalPaid      int
+	Case              sirius.Case
+	Payments          []sirius.Payment
+	PaymentSources    []sirius.RefDataItem
+	ReferenceTypes 	  []sirius.RefDataItem
+	User              sirius.User
```

Comment:
I don't think you need the user in the template data?
```suggestion
	User              sirius.User
```